ocaml/idl/datamodel.ml

```diff
@@ -6066,6 +6067,7 @@ let vif =
          field ~ty:vif_ipv6_configuration_mode ~in_product_since:rel_dundee ~qualifier:DynamicRO "ipv6_configuration_mode" "Determines whether IPv6 addresses are configured on the VIF" ~default_value:(Some (VEnum "None"));
          field ~ty:(Set (String)) ~in_product_since:rel_dundee ~qualifier:DynamicRO "ipv6_addresses" "IPv6 addresses in CIDR format" ~default_value:(Some (VSet []));
          field ~ty:String ~in_product_since:rel_dundee ~qualifier:DynamicRO "ipv6_gateway" "IPv6 gateway (the empty string means that no gateway is set)" ~default_value:(Some (VString ""));
+         field ~ty:(Ref _pci) ~in_product_since:rel_jura ~qualifier:DynamicRO "attached_pci" "pci of which this vif attached (sriov vif)" ~default_value:(Some (VRef null_ref));
```

Comment:
The documentation strings needs attention; the meaning is not clear, It could be either of these:
* pci where this VIF is attached?
* pci that is attached to this VIF?

Reply:
Will update. Actual meaning of this field is the pci where this VIF attached